### PR TITLE
Rename Panel/Fold/Section -> PlotlyPanel/Fold/Section/

### PR DESCRIPTION
**Pull request by VeraZab**
fixes: #403

### CODE REVIEW

File: src/components/containers/PlotlyFold.js [hunk 1]
```diff
@@ -149,13 +154,37 @@ Fold.propTypes = {
...
+Fold.defaultProps = {
+  forceVisibility: true,
+};
+
+Fold.propTypes = {
+  children: PropTypes.node,
+  className: PropTypes.string,
+  folded: PropTypes.bool,
+  toggleFold: PropTypes.func,
+  hideHeader: PropTypes.bool,
```
Comment (VeraZab) [inline src/components/containers/PlotlyFold.js]: I removed the isRequired here...
BUT, I am testing in my tests that those props are there for `<Fold/>`s
I don't know why those console.errors were occurring..

File: src/components/containers/PlotlyPanel.js [hunk 2]
```diff
@@ -123,23 +124,23 @@ class Panel extends Component {
   }
 }
 
-Panel.plotly_editor_traits = {no_visibility_forcing: true};
+PlotlyPanel.plotly_editor_traits = {no_visibility_forcing: true};
 
-Panel.propTypes = {
+PlotlyPanel.propTypes = {
   children: PropTypes.node,
   addAction: PropTypes.object,
```
Comment (VeraZab) [inline src/components/containers/PlotlyPanel.js]: I left the addAction on `<Panel/>` maybe devs would like to add some non context related addActions..
Doesn't have to be disabled no?

File: src/components/containers/SectionHeader.js [hunk 3]
```diff
@@ -1,16 +0,0 @@
...
-      <div className="section__heading">
-        <div className="section__heading__text">{this.props.name}</div>
-      </div>
-    );
-  }
-}
-
-SectionHeader.propTypes = {
-  name: PropTypes.string,
-};
```
Comment (VeraZab) [inline src/components/containers/SectionHeader.js]: we don't need this anymore.. also Ben didn't use it in his code :)

File: src/components/fields/Info.js [hunk 4]
```diff
@@ -3,7 +3,11 @@ import React, {Component} from 'react';
 
 export default class Info extends Component {
   render() {
-    return <Field {...this.props}>{this.props.children}</Field>;
+    return (
+      <Field {...this.props}>
+        <div className="js-test-info">{this.props.children}</div>
+      </Field>
```
Comment (VeraZab) [inline src/components/fields/Info.js]: If I add the class on Field, it gives me 2 js-test-info classes, one on the localized wrapper one on Field

**Comment by VeraZab** [comment]
@nicolaskruchten would you take a 👀

File: src/components/containers/PlotlyPanel.js [hunk 5]
```diff
@@ -123,18 +123,20 @@ export class Panel extends Component {
...
 
-Panel.defaultProps = {
+UnlocalizedPanel.defaultProps = {
   showExpandCollapse: true,
 };
 
-export default class PlotlyPanel extends Panel {}
+export const Panel = localize(UnlocalizedPanel);
+
+export default class PlotlyPanel extends UnlocalizedPanel {}
```
Comment (nicolaskruchten) [inline src/components/containers/PlotlyPanel.js]: that's weird... the Panel and Section were not localized but the Fold was ???

**Comment by nicolaskruchten** [comment]
💃  thanks for putting up with these requirements and changes :)

**Pull request merged**